doc/modules/classes.rst

```diff
@@ -1602,7 +1602,6 @@ Plotting
    utils.gen_batches
    utils.gen_even_slices
    utils.graph.single_source_shortest_path_length
-   utils.graph.graph_shortest_path
```

Comment:
Since we are deprecating, I would let this public until the end of the deprecation.

Reply:
Actually, I don't even see it in the documentation currently